pkg/controllers/user/rbac/handler_base.go

```diff
@@ -472,7 +412,12 @@ func crbRoleSubjectKeys(roleName string, subjects []rbacv1.Subject) []string {
 }
 
 func rbRoleSubjectKey(roleName string, subject rbacv1.Subject) string {
-	return subject.Kind + " " + subject.Name + " Role " + roleName
+	prefix := subject.Kind + " "
```

Comment:
This function is used in 5 places. None of them are broken by this change to its logic?